vars/spinInfra.groovy

```diff
@@ -62,7 +62,7 @@ def call(Map<String, ?> params) {
 
         def tags = [environment: Environment.toTagName(config.environment), changeUrl: changeUrl, managedBy: teamName, BuiltFrom: builtFrom, contactSlackChannel: contactSlackChannel, application: env.TEAM_APPLICATION_TAG, businessArea: env.BUSINESS_AREA_TAG]
 
-        if (new Environment(env).sandbox == config.environment || config.environment == "sbox") {
+        if (new Environment(env).sandbox == config.environment || Environment.toTagName(config.environment) == "sandbox" || config.environment == "sbox") {
```

Comment:
from what I can tell this whole condition can be replaced by what you just added?
```suggestion
        if (Environment.toTagName(config.environment) == "sandbox") {
```

Reply:
I think you might be right